Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -643,6 +707,103 @@ def get_network_info(
 
         return self._http_request("GET", "network", params=request_params)
 
+    def update_rpz_rule(
+        self,
+        reference_id: str | None,
+        rule_type: str | None,
```

Comment:
no need to add | None since it's required in the yml and will never have the value None
```suggestion
        rule_type: str,
```